garbage-collect/handler.go

```diff
@@ -42,6 +44,12 @@ func Handle(req []byte) string {
 		log.Fatal(err)
 	}
 
+	auditEvent := sdk.AuditEvent{
```

Comment:
I think we could avoid this pre-delete message. It may end-up being too spammy.

Reply:
Ok I will remove this